pkg/util/validation/validation.go

```diff
@@ -37,7 +37,7 @@ func ValidatePlacement(placement policyv1alpha1.Placement, fldPath *field.Path)
 	return allErrs
 }
 
-// ValidateClusterAffinity validates a PropagationSpec before creation or update.
+// ValidateClusterAffinity validates a clusterAffinity before creation or update.
```

Comment:
nice finding.

Reply:
https://github.com/karmada-io/karmada/blob/ea497819f0ee883c8f46fbb390ac540df27f2b0b/pkg/util/validation/validation.go#L26

https://github.com/karmada-io/karmada/blob/ea497819f0ee883c8f46fbb390ac540df27f2b0b/pkg/util/validation/validation.go#L54

Two more places, could you please fix them as well?